### PR TITLE
upgrade to lbry.go v2.4.0

### PR DESCRIPTION
**Pull request by sayplastic**
This includes updated references to the latest `lbry.go` version.

I also downgraded some excessive INFO-level logging calls to DEBUG.

### CODE REVIEW

File: peer/store.go [hunk 1]
```diff
@@ -12,10 +14,19 @@ type Store struct {
...
+}
+
 // NewStore makes a new peer store.
-func NewStore(clientAddress string) *Store {
-	c := &Client{}
-	err := c.Connect(clientAddress)
+func NewStore(opts StoreOpts) *Store {
+	if opts.Timeout == 0 {
+		opts.Timeout = time.Second * 5
+	}
```
Comment (lyoshenka) [inline peer/store.go]: doing this is not a good idea. just set the 0 as the Client timeout. that already signals the Client to use a default.

if you do this here, not only is this duplicating code, but now if the Client's default changes, you'll be stuck on 5 seconds here (unless this was your intention???)

File: peer/store.go [hunk 2]
```diff
@@ -12,10 +14,19 @@ type Store struct {
 	connErr error
 }
 
+// StoreOpts allows to set options for a new Store.
+type StoreOpts struct {
```
Comment (lyoshenka) [inline peer/store.go]: i dont love this design here, but maybe there's no better way to do optional arguments in Go?

no need to spend much time on improving this. just thinking out loud.

File: reflector/uploader.go [hunk 3]
```diff
@@ -167,17 +164,17 @@ func (u *Uploader) uploadBlob(filepath string) (err error) {
 	}
 
 	if IsValidJSON(blob) {
-		log.Debugf("Uploading SD blob %s", hash)
+		log.Debugf("uploading SD blob %s", hash)
```
Comment (lyoshenka) [inline reflector/uploader.go]: :heart:

File: peer/store.go (hunk 2 above)
Comment (sayplastic) [reply peer/store.go]: Seems to be the default method of accepting optional parameters for functions, I see it all the time in 3rd party code too. I personally would've preferred pattern matching (a la Elixir) or optional arguments (Python) but I simply don't see a better way in Go.

**Pull request merged**